editors/HexFiend/Portfile

```diff
@@ -51,9 +47,9 @@ destroot {
     xinstall -m 644 ${worksrcpath}/License.txt ${destroot}${prefix}/share/doc/${name}
 }
 
-if {${os.platform} ne "darwin" || ${os.major} < 12} {
+if {${os.platform} ne "darwin" || ${os.major} < 18} {
     pre-fetch {
-        ui_error "${subport} @${version} only works with OS X 10.8 Mountain Lion or later."
-        return -code error "incompatible OS X version"
+        ui_error "${subport} @${version} only works with macOS 10.14 Mojave or later."
```

Comment:
On this line, should it be `${subport}` or something else now that there are no subports? Or does it still work like this?

Reply:
`$subport` will equal `$name`, but it will still be set, so no need to change that.